### PR TITLE
feat: add a optional param `silence` to control whether print the warni…

### PR DESCRIPTION
**Pull request by dancon**
add a optional param silence to control whether print the warning when there is already a content prop.

### CODE REVIEW

**Comment by springuper** [comment]
Hey, @dancon, could you tell me why do you need this option?

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 npm-debug.log
+package-lock.json
```
Comment (springuper) [inline .gitignore]: please add newline at the end of the file.

File: README.md [hunk 2]
```diff
@@ -8,21 +8,23 @@ Automatically append `content` property for [viewport-units-buggyfill](https://g
 ## Install
 
 ```bash
-$ npm install postcss-viewport-units
+  npm install postcss-viewport-units
```
Comment (springuper) [inline README.md]: prefer to keep the shell prompt, it indicates that this is a shell command.

Comment (dancon) [reply README.md]: ok ~~

**Comment by dancon** [comment]
> Hey, @dancon, could you tell me why do you need this option?

I have committed an issue, you can check it [here](https://github.com/springuper/postcss-viewport-units/issues/9)

**Comment by dancon** [comment]
@springuper Is there any problems? why not accept the PR. The reason that I need the option is [here](https://github.com/springuper/postcss-viewport-units/issues/9)

**Comment by DevinZhong** [comment]
well done. i like this feature. @dancon

**Comment by DevinZhong** [comment]
the `filterRule` option can solve my problem, you may have a try. @dancon https://github.com/springuper/postcss-viewport-units/issues/9#issuecomment-426872132

File: README.md [hunk 3]
```diff
@@ -8,21 +8,23 @@ Automatically append `content` property for [viewport-units-buggyfill](https://g
 ## Install
 
 ```bash
-$ npm install postcss-viewport-units
+> $ npm install postcss-viewport-units
```
Comment (springuper) [inline README.md:11]: `$` is the prompt, no need for `>`. i'll fix it after merging your PR.

**Review by springuper — APPROVED** [review]
LGTM.

**Comment by springuper** [comment]
@dancon thanks for contribution. sorry for the late.

**Comment by springuper** [comment]
postcss-viewport-units@0.1.5 released with this change. @DevinZhong yes, `filterRule` can also help, but this flag is more straightforward.

**Pull request merged**